webextensions/_locales/fr/messages.json

```diff
@@ -133,250 +133,254 @@
 
-  "bookmarkDialog_title":    { "message": "Titre:" },
-  "bookmarkDialog_url":      { "message": "Adresse:" },
-  "bookmarkDialog_parentId": { "message": "Dossier:" },
+  "bookmarkDialog_title":    { "message": "Titre :" },
+  "bookmarkDialog_url":      { "message": "Adresse :" },
+  "bookmarkDialog_parentId": { "message": "Dossier :" },
   "bookmarkDialog_accept":   { "message": "Sauvegarder" },
   "bookmarkDialog_cancel":   { "message": "Annuler" },
 
   "bookmarkFolderChooser_unspecified":   { "message": "(non spécifié)" },
-  "bookmarkFolderChooser_blank":         { "message": "(pas de nom)" },
+  "bookmarkFolderChooser_blank":         { "message": "(sans titre)" },
   "bookmarkFolderChooser_useThisFolder": { "message": "Utiliser ce dossier" },
 
-  "message_startup_description_1": { "message": "Tree Style Tab's vertical tab bar is available as one of choosable sidebar panels. If you don't see it yet, hit the " },
-  "message_startup_description_key": { "message": "touche \"F1\"" },
-  "message_startup_description_2": { "message": " ou cliquer sur " },
-  "message_startup_description_3": { "message": " button in the toolbar, to activate the sidebar panel." },
+  "message_startup_description_1": { "message": "La barre d'onglets de Tree Style Tab est l'un des choix de panneau latéral disponibles. Si vous ne le voyez pas encore, appuyez sur la " },
+  "message_startup_description_key": { "message": "touche « F1 »" },
+  "message_startup_description_2": { "message": " ou cliquez sur le bouton " },
+  "message_startup_description_3": { "message": " dans la barre d'outils, pour activer le panneau latéral." },
 
-  "message_startup_requestPermissions_description": { "message": "⚠Il y a quelques fonctionnalités qui requierent des permissions supplémentaires. Si vous voulez les utiliser, acceptez les permissions manuellement." },
-  "message_startup_requestPermissions_bookmarks": { "message": "\"Marquer cette arborescence\" dans le menu contextuel sur les onglets (requiert l'accès aux marque-page)" },
-  "message_startup_requestPermissions_allUrls":   { "message": "Don't expand collapsed tree and skip collapsed descendants, while switching focus of tabs via keyboard shortcuts (requires script injection to webpages for detection of keyboard operations)" },
+  "message_startup_requestPermissions_description": { "message": "⚠Quelques fonctionnalités requièrent des permissions supplémentaires. Si vous voulez les utiliser, acceptez les permissions manuellement." },
+  "message_startup_requestPermissions_bookmarks": { "message": "« Ajouter cet arbre aux marque-pages » dans le menu contextuel sur les onglets (requiert l'accès aux marque-pages)" },
+  "message_startup_requestPermissions_allUrls":   { "message": "Ne pas déplier les arbres repliés et ignorer les descendants repliés lorsque vous changez d'onglet courant via des raccourcis claviers (nécessite une injection de script dans les pages web pour détecter les appuis de touche)" },
```

Comment:
Maybe `Ne pas ouvrir`?

Reply:
Like one of your comment above, I think this one is logical and shouldn’t be changed.